### ISSUE TITLE
Screen readers ignore label.visuallyhidden

### ISSUE THREAD

**Issue by therazor** [h5bp/html5-boilerplate#1242]
While `.visuallyhidden` works generally well with screenreaders, when applied to a label and focusing the corresponding input, the screenreader fails to announce its description, at least on ffox 12 with nvda - chrome dev + nvda works well however.
I have seen another report of this happening on [a jquery project](http://blog.ginader.de/dev/jquery/HTML5-placeholder-polyfill/index-chromeish.html).

Please see [this jsfiddle](http://jsfiddle.net/Razor/gS5up/1/) for a demonstration.
